packages/st2mistral/debian/conffiles

```diff
@@ -0,0 +1 @@
+/etc/logrotate.d/mistral
```

Comment:
For `deb` the expected behavior is different: `/etc/logroate.d/mistral` is not removed on `apt-get remove st2mistral`, but it's removed on `apt-get purge st2mistral`, which removes the package with configs.